go.mod

```diff
@@ -120,9 +126,11 @@ require (
 	github.com/hashicorp/go-cleanhttp v0.5.1 // indirect
 	github.com/hashicorp/go-retryablehttp v0.7.1 // indirect
 	github.com/jmespath/go-jmespath v0.4.0 // indirect
+	github.com/kylelemons/godebug v1.1.0 // indirect
```

Comment:
This module does a lot of reflection etc, but looks to be used only to provide a "pretty" verbose error; https://github.com/AzureAD/microsoft-authentication-library-for-go/blob/751ec246d696ada22aee7a7c3bfd7a097b248250/apps/errors/errors.go#L77

I'm not very familiar with this package, but it looks to be managed by a single-maintainer, and not very widely used, and it lists github.com/google/go-cmp and github.com/davecgh/go-spew as possible alternatives (both of which _are_ already used, but even those only for tests).

Happy to hear input on that from others, but given that the functionality of that package (from the looks of it) doesn't seem to be _essential_, I'm wondering if there would be ways to remove it (could be through a build-tag perhaps).

Reply:
The package definitely looks like it is widely used including several very widely used projects like prometheus-operator, istio etc.. Also, the other packages mentioned (gp-cmp, go-spew) also don't look like they are maintained much which is understandable considering how light the functionality they provide is. I really don't see this as a problem unless someone has a strong opinion on why this particular package is bad enough to make an exception by allowing it to be skipped somehow

Reply:
A difference here is that most of those don't use this in _production_ code, only in tests; e.g. https://cs.github.com/prometheus-operator/prometheus-operator?q=github.com%2Fkylelemons%2Fgodebug
https://cs.github.com/istio/istio?q=github.com%2Fkylelemons%2Fgodebug

Not a strong blocker for me, but let me "unresolved" the conversation to keep the converesation visible for future reference